cucumber/features/fabric-gateways.feature

```diff
@@ -26,7 +26,7 @@ Feature: Fabric Gateways
 
     @otherFabric
     Scenario: Create another gateway
-        When I create a gateway 'myGateway'
+        When I create a gateway 'myGateway' from a 'profile'
```

Comment:
showEnvironmentQuickPick presented an empty list when trying to add a gateway from an environment (when there were no other envs other than the Local Fabric)